pkg/operator/deploy/deploy_test.go

```diff
@@ -41,8 +41,8 @@ func TestCheckIngressIP(t *testing.T) {
 					},
 				}
 			},
-			want:    "1.2.3.4",
-			wantErr: nil,
+			want:       "1.2.3.4",
+			wantErrMsg: "",
```

Comment:
nit: we can omit setting `wantErrMsg` to `""`

Reply:
same for `want` down below

Reply:
Done. Tks!